src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_process_net_command_json.py

```diff
@@ -184,6 +186,85 @@ def on_attach_request(self, py_db, request):
         response = pydevd_base_schema.build_response(request)
         return NetCommand(CMD_RETURN, 0, response.to_dict(), is_json=True)
 
+    def on_pause_request(self, py_db, request):
+        '''
+        :param PauseRequest request:
+        '''
+        arguments = request.arguments  # : :type arguments: PauseArguments
+        thread_id = arguments.threadId
+
+        self.api.request_suspend_thread(py_db, thread_id=thread_id)
+
+        response = pydevd_base_schema.build_response(request)
+        return NetCommand(CMD_RETURN, 0, response.to_dict(), is_json=True)
+
+    def on_continue_request(self, py_db, request):
+        '''
+        :param ContinueRequest request:
+        '''
+        arguments = request.arguments  # : :type arguments: ContinueArguments
+        thread_id = arguments.threadId
+
+        self.api.request_resume_thread(thread_id)
+
+        body = {'allThreadsContinued': True} if thread_id == '*' else {}
+        response = pydevd_base_schema.build_response(request, kwargs={'body': body})
+        return NetCommand(CMD_RETURN, 0, response.to_dict(), is_json=True)
+
+    def on_next_request(self, py_db, request):
+        '''
+        :param NextRequest request:
+        '''
+        arguments = request.arguments  # : :type arguments: NextArguments
+        thread_id = arguments.threadId
+
+        if thread_id.startswith('*'):
```

Comment:
This is going to break once we switch thread IDs to numbers, as the spec requires. Perhaps make it a separate message for step-over-my-code? We could also add a custom field to the request - I think that's allowed.

Reply:
I agree. @fabioz, do you have any preference here?

Reply:
I think `pydevd` should respect the `just-my-code` global setting and receive just a regular `next` request with the thread id without the `*` prefix (the settings are already set in `_set_debug_options`... it sets the value at `self.api.set_use_libraries_filter`, so, a `self.api.get_use_libraries_filter` should be done and that setting should be used to decide which internal command should be used).